kuzzleio/kuzzle-plugin-cluster
# KZL-998- persist strategies in redis

## What does this PR do ?

This PR fixes the case where some strategies could be lost across a cluster of Kuzzle nodes if one or more of them came to not respond by persisting the strategies in Redis.

### How should this be manually tested?

- Add this small plugin to your cluster: https://gist.github.com/benoitvidis/a5d9bd592a261d7b903209e73abed59d
- Check the `dummy` strategy does not exist by trying to log in
- Call http://localhost:7512/_plugin/test/do/bar
- Check the `dummy` strategy is known on all nodes
- Call http://localhost:7512/_plugin/test/do/foo
- Check the `dummy` strategy has been deleted from all nodes
